ops/testing.py

```diff
@@ -59,39 +58,31 @@
 from ops import charm, framework, model, pebble, storage
 from ops._private import yaml
 from ops.charm import CharmBase, CharmMeta, RelationRole
-from ops.model import Container, RelationNotFoundError
+from ops.model import Container, RelationNotFoundError, _ConfigOption, _NetworkDict
 from ops.pebble import ExecProcess
 
-if TYPE_CHECKING:
-    from ops.model import _NetworkDict
-
-    ReadableBuffer = Union[bytes, str, StringIO, BytesIO, BinaryIO]
-    _StringOrPath = Union[str, pathlib.PurePosixPath, pathlib.Path]
-    _FileKwargs = TypedDict('_FileKwargs', {
-        'permissions': Optional[int],
-        'last_modified': datetime.datetime,
-        'user_id': Optional[int],
-        'user': Optional[str],
-        'group_id': Optional[int],
-        'group': Optional[str],
-    })
-
-    _RelationEntities = TypedDict('_RelationEntities', {
-        'app': str,
-        'units': List[str]
-    })
-
-    _ConfigOption = TypedDict('_ConfigOption', {
-        'type': Literal['string', 'int', 'float', 'boolean'],
-        'description': str,
-        'default': Union[str, int, float, bool],
-    })
-    _StatusName = Literal['unknown', 'blocked', 'active', 'maintenance', 'waiting']
-    _RawStatus = TypedDict('_RawStatus', {
-        'status': _StatusName,
-        'message': str,
-    })
-    RawConfig = TypedDict("RawConfig", {'options': Dict[str, _ConfigOption]})
+ReadableBuffer = Union[bytes, str, StringIO, BytesIO, BinaryIO]
+_StringOrPath = Union[str, pathlib.PurePosixPath, pathlib.Path]
+_FileKwargs = TypedDict('_FileKwargs', {
+    'permissions': Optional[int],
+    'last_modified': datetime.datetime,
+    'user_id': Optional[int],
+    'user': Optional[str],
+    'group_id': Optional[int],
+    'group': Optional[str],
+})
+
+_RelationEntities = TypedDict('_RelationEntities', {
+    'app': str,
+    'units': List[str]
+})
+
+_StatusName = Literal['unknown', 'blocked', 'active', 'maintenance', 'waiting']
+_RawStatus = TypedDict('_RawStatus', {
+    'status': _StatusName,
+    'message': str,
+})
+RawConfig = TypedDict("RawConfig", {'options': Dict[str, _ConfigOption]})
```

Comment:
I was looking at the [API docs](https://ops--1034.org.readthedocs.build/en/1034/#ops.testing.RawConfig), and I noticed this accidentally became part of the public API. Can it be underscore-prefixed?